internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -200,6 +205,15 @@ func (opts *Opts) Run() error {
 	return nil
 }
 
+func (opts *Opts) clusterError(err error) error {
+	var target *atlas.ErrorResponse
+	if errors.As(err, &target) && target.ErrorCode == "CANNOT_CREATE_FREE_CLUSTER_VIA_PUBLIC_API" {
+		return fmt.Errorf("%w",
+			ErrFreeClusterAlreadyExists)
```

Comment:
wrapping seems unnecessary here
```suggestion
		return ErrFreeClusterAlreadyExists
```